experiments/20160805-mutagenesis/20160902-mutagenesis-2.md

```diff
@@ -78,3 +78,32 @@
 | 72              | 2 min           | times    |
 | 72              | 7 min           | 1        |
 | 4               | hold            |          |
+
+### Gel Electrophoresis
+
+Gel Parameters
+- 1% (w/v), 50 mL TBE
+- 150 volts, 35 min
+
+Gel Order
+
+| Ladder | P2 | PW | PT | A1 | A2 | AW | AT | Ladder |
+|--------|----|----|----|----|----|----|----|--------|
+
+![gel](./20160903-mutagenesis-2.jpg)
+
+## Analysis
+
+**Expected Sizes**
+
+| Fragment        | expected size | successful?|
+|-----------------|---------------|------------|
+| A1: PB2 Part 1  | 1600          | No...      |
+| A2: PB2 Part 2  | 820           | Faint      |
+| P2: pCI Part 2  | 1374          | YES       |
+
+- **A1**: The two brightest bands are at smaller sizes than expected, indicating fragmentation. A much fainter band between the 1.5 and 2.0 reference bands could represent DNA of the expected size.
```

Comment:
Looking at the gel, I think there may be another interpretation of the smaller bands - mispriming? Agree that the fainter band could be the one we want. Give the gel extraction a shot?